spec/commands/csv_data_upload_spec.rb

```diff
@@ -49,14 +49,53 @@
       legislations_csv,
       new_records: 2, not_changed_records: 0, rows: 2, updated_records: 0
     )
+
+    legislation = Legislation.find_by(title: 'Climate Law')
+
+    expect(legislation.legislation_type).to eq('legislative')
+    expect(legislation.description).to eq('Description')
+    expect(legislation.document_types_list).to include('Law')
+    expect(legislation.visibility_status).to eq('pending')
+    expect(legislation.geography.iso).to eq('POL')
+    expect(legislation.frameworks.size).to eq(2)
+    expect(legislation.frameworks_list).to include('Mitigation', 'Adaptation')
+    expect(legislation.keywords.size).to eq(2)
+    expect(legislation.keywords_list).to include('keyword1', 'keyword3')
+    expect(legislation.natural_hazards.size).to eq(2)
+    expect(legislation.natural_hazards_list).to include('sharkinados', 'flooding')
   end
 
   it 'imports CSV files with Litigation data' do
+    legislation1 = create(:legislation)
+    legislation2 = create(:legislation)
+    updated_litigation = create(:litigation)
+
+    csv_content = <<-CSV
```

Comment:
I understand that this needs to be dynamic here, but can we still keep this in `let`-vars?

Reply:
I will see if that will be more readable, maybe if we split those tests to separate files to test only importer services.

Reply:
yeah, maybe we can move to `let`s just things like `legislation2` , `updated_litigation`?

Reply:
Hmm, moving those values to `let` globally would make test less readable in this case. Everything declared directly in test makes things easier when the test file is too big. There should be either `context` or separate file, but I don't think we have time to refactor tests much.

Reply:
Lol, I should have picked MiniTest instead of RSpec, fewer things to remember. Minimalism FTW!